deluge/ui/gtkui/dialogs.py

```diff
@@ -328,21 +328,21 @@ def __init__(self, header, text="", unit_text="", icon=None, default=0, parent=N
             raise TypeError("default value needs to be an int or float")
 
         if not icon:
-            icon = gtk.STOCK_DIALOG_INFO
+            icon = Gtk.STOCK_DIALOG_INFO
 
         super(OtherDialog, self).__init__(
             header,
             text,
             icon,
-            (gtk.STOCK_CANCEL, gtk.RESPONSE_CANCEL, gtk.STOCK_APPLY, gtk.RESPONSE_OK),
+            (Gtk.STOCK_CANCEL, Gtk.ResponseType.CANCEL, Gtk.STOCK_APPLY, Gtk.ResponseType.OK),
             parent)
 
-        hbox = gtk.HBox(spacing=5)
-        alignment_spacer = gtk.Alignment()
-        hbox.pack_start(alignment_spacer)
-        alignment_spin = gtk.Alignment(1, 0.5, 1, 1)
-        adjustment_spin = gtk.Adjustment(value=-1, lower=-1, upper=2097151, step_incr=1, page_incr=10)
-        self.spinbutton = gtk.SpinButton(adjustment_spin)
+        hbox = Gtk.HBox(spacing=5)
+        alignment_spacer = Gtk.Alignment.new(1, 0.5, 1, 1)
```

Comment:
Why these values? It should be `0, 0, 0, 0` looking at the default for [gtk.Alignment](http://www.pygtk.org/pygtk2reference/class-gtkalignment.html#constructor-gtkalignment).

Also it appears [gtk-alignment-new](https://developer.gnome.org/gtk3/stable/GtkAlignment.html#gtk-alignment-new) is deprecated.